common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -669,19 +669,29 @@ def get_transcripts_info(self, is_bumper=False):
         """
         Returns a transcript dictionary for the video.
         """
+        transcripts = self.transcripts
+        sub = self.sub
         if is_bumper:
             transcripts = copy.deepcopy(get_bumper_settings(self).get('transcripts', {}))
             sub = transcripts.pop("en", "")
-        else:
```

Comment:
@noraiz-anwar Thanks. It's a very good and precise solution. I will update it according to your suggestion.